src/main/java/com/auth0/jwk/Jwk.java

```diff
@@ -61,7 +61,8 @@ static Jwk fromValues(Map<String, Object> map) {
         String kty = (String) values.remove("kty");
         String alg = (String) values.remove("alg");
         String use = (String) values.remove("use");
-        String keyOps = (String) values.remove("key_ops");
+        @SuppressWarnings("unchecked")
+        List<String> keyOps = (List<String>) values.remove("key_ops");
```

Comment:
Same comment goes here. We should still support the old/wrong type. Doing this cast will fail. I suggest you add some tests to assert that both behaviors work as expected.

Reply:
Thank you for your feedback! For this one, I can't think of an eloquent way to address this without writing a conditional to check the object type during runtime before parsing. Do you have any particular preferences on how you'd approach this?